src/main/java/io/singularitynet/Server/VereyaModServer.java

```diff
@@ -76,6 +80,18 @@ public void onInitialize() {
                 this.stateMachine.queueStateChange(ServerState.WAITING_FOR_MOD_READY);
             }
         });
+
+        ServerPlayConnectionEvents.JOIN.register((handler, sender, server) -> {
+            ServerPlayerEntity player = handler.player;
+            // Grant op status to the player so the commands will work for sure
+            MinecraftServer minecraftServer = player.getServer();
+            if (minecraftServer != null) {
+                GameProfile gameProfile = player.getGameProfile();
+                OperatorEntry operatorEntry = new OperatorEntry(gameProfile, 4, true);
+                minecraftServer.getPlayerManager().getOpList().add(operatorEntry);
```

Comment:
I think we should limit this operation only for bots, maybe here https://github.com/trueagi-io/Vereya/blob/2cbae3a084b2187bd0abe9fdd2438477bd6278d4/src/main/java/io/singularitynet/Server/ServerStateMachine.java#L773C52-L773C58

Reply:
So I should just put this code in the section you linked to. Is that correct?

Reply:
yes